### DISCUSSION TITLE
MUI X License Check Air-Gapped Computer

### DISCUSSION THREAD

**Discussion by jmuncaster** [mui/mui-x#2921] [Archive]
I have an application that runs on a local sever disconnected from the web. What happens if the license check fails due to lack of network access?

**Comment by m4theushw** [comment]
You don't need to have network access to use DataGridPro. The license verification doesn't make requests: https://mui.com/x/introduction/licensing/#license-key-security

**Reply by jmuncaster** [reply]
Thank you!
